package-lock.json

```diff
@@ -1700,6 +1702,15 @@
       "resolved": "https://registry.npmjs.org/client-only/-/client-only-0.0.1.tgz",
       "integrity": "sha512-IV3Ou0jSMzZrd3pZ48nLkT9DA7Ag1pnPzaiQhpW7c3RbcqqzvzzVu+L8gfqMp/8IM2MQtSiqaCxrrcfu8I8rMA=="
     },
+    "node_modules/clsx": {
```

Comment:
why do we need this dependency?  any specific reason?

Reply:
i have used clsx in cn function to add dynamic classes.

Reply:
i have used clsx in cn function to add dynamic classes.

Reply:
i have used clsx in cn function to add dynamic classes.